test/com/amazon/ion/impl/lite/BaseIonSequenceLiteSublistTestCase.java

```diff
@@ -25,12 +25,13 @@
 import java.util.Iterator;
 import java.util.List;
 import java.util.ListIterator;
-import org.junit.Test;
+
 import com.amazon.ion.ContainedValueException;
 import com.amazon.ion.IonInt;
 import com.amazon.ion.IonSequence;
 import com.amazon.ion.IonSystem;
 import com.amazon.ion.IonValue;
+import org.junit.Test;
 import com.amazon.ion.system.IonSystemBuilder;
```

Comment:
Are we changing import order style?

Reply:
My IntelliJ keeps reseting its code style every time I import a new project for some reason I can't fathom.